spread/project.go

```diff
@@ -854,6 +854,16 @@ func (p *Project) Jobs(options *Options) ([]*Job, error) {
 		return nil, fmt.Errorf("remote project path must be absolute and not /: %s", p.RemotePath)
 	}
 
+	// In case the number of workers set in the options is bigger than 0,
```

Comment:
This comment is slightly misleading now.

Reply:
done